ankaios_sdk/_components/control_interface.py

```diff
@@ -278,14 +278,13 @@ def _read_from_control_interface(self) -> None:
                         break
 
                 if not varint_buffer:
-                    if self._state != ControlInterfaceState.AGENT_DISCONNECTED:
-                        self.change_state(
-                            ControlInterfaceState.AGENT_DISCONNECTED)
+                    self.change_state(
+                        ControlInterfaceState.AGENT_DISCONNECTED)
                     self._logger.error(
```

Comment:
Since we have a handling now, we could go for a warning and not an error.

Reply:
Done.